app/jinja_filters.py

```diff
@@ -23,6 +23,7 @@ def format_number(value):
 
 
 def get_formatted_address(address_fields):
+    address_fields.pop("uprn", None)
```

Comment:
Is `uprn` guaranteed to be present in the address_fields dict?

Reply:
It isn't guaranteed if you are not using the lookup.

Reply:
It's a very very minor comment in any case - just if I see the above I assume that it will legitimately be missing as a key in some scenarios

Reply:
No - when the lookup isn't used the `uprn` field is not stored in the address answer in the answer store.